MatteoOrlandini/PDF-Unlocker
# Added Ubuntu-specific pikepdf Installation Instructions

### Changes Made
I have updated the README.md to include clear instructions for Ubuntu users on how to install `pikepdf` for Python 3.10 using a virtual environment. This addresses the issue of `pikepdf` not being available yet in the standard Ubuntu repositories.

**Steps to Verify:**
1. Clone this repository.
2. Create a virtual environment and activate it.
3. Follow the provided instructions to install `pikepdf`.
4. Verify that `pikepdf` is working correctly within the virtual environment.

**Checklist**
- [x] README.md updated with clear installation instructions.
- [x] Tested on Ubuntu 20.04 with Python 3.10.

Please review and merge this pull request to help Ubuntu users easily install `pikepdf` for Python 3.10.
